crates/turborepo-auth/src/server/login_server.rs

```diff
@@ -56,6 +51,7 @@ impl LoginServer for DefaultLoginServer {
         Ok(axum_server::bind(addr)
             .handle(handle)
             .serve(app.into_make_service())
-            .await?)
+            .await
+            .expect("failed to start one-shot server"))
```

Comment:
If we fail to start the one-shot server, it's a fatal error with nothing that we can do. A panic here is acceptable imo